examples/js/effects/VREffect.js

```diff
@@ -72,17 +90,30 @@ THREE.VREffect = function ( renderer, onError ) {
 	var isPresenting = false;
 
 	var canvas = renderer.domElement;
-	var fullscreenchange = canvas.mozRequestFullScreen ? 'mozfullscreenchange' : 'webkitfullscreenchange';
+	var fullscreenchange;
+	var fullscreenElement;
 
-	document.addEventListener( fullscreenchange, function () {
+	if ( canvas.requestFullscreen ) {
 
-		if ( vrHMD && deprecatedAPI ) {
+		fullscreenchange = 'fullscreenchange';
```

Comment:
latest Firefox Nightly builds support unprefixed Fullscreen API, so this gets rid of deprecation warnings